lib/trader-sdk/test/indicators/bb-percent-b-mfi.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { describe } from 'mocha';
+import { expect } from 'chai';
+import _ from 'lodash';
+import { FourteenDayPeriodWalksTheLine, FourteenDayPeriod } from '../../mocks/mfi.mock';
+import { BPercentMfiIndicator } from '../../lib/indicators'
+import { BBHighBPercent, BBSecond } from '../../mocks/BB.mock';
+import Sinon from 'sinon';
+const { BollingerBands, MFI } = require('technicalindicators');
+
+describe('B% MFI Indicator::', function () {
+
+    let sandbox: Sinon.SinonSandbox;
+    let stubs: any = {};
+
+    beforeEach(() => {
+        sandbox = Sinon.createSandbox();
+        //Upper test setup
+        stubs.BBStub = sandbox.stub(BollingerBands, 'calculate').returns(BBHighBPercent);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/dills122/trader-tools/pullRequest?prid=8102010)
```suggestion
        stubs.BBStub = sandbox.stub(BollingerBands, "calculate").returns(BBHighBPercent);
```